### PR TITLE
Add GH templates

### PR DESCRIPTION
**Pull request by helderburato**
## Description

It will enable templates for new issues and PR's.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Issue:
```
Comment (obetomuniz) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: What about creating an approach that makes issue and test declaration optional or less opinionated? I mean, for non-code PRs like documentation PRs, improvement or integrations suggestions, etc.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### To Reproduce
+
+(Write your steps such as:)
+
+1. Step 1...
+1. Step 2...
+1. Step 3...
+
+### Browsers Affected
```
Comment (obetomuniz) [inline .github/ISSUE_TEMPLATE/bug_report.md:51]: I would ask for the Operational System too. WDYT?

**Review by obetomuniz — APPROVED** [review]
Just added a few nits. Overall, looks great. Thanks for bringing such improvement.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (helderburato) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I agree with you, do you have a suggestion for this?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (helderburato) [reply .github/ISSUE_TEMPLATE/bug_report.md:51]: I will update with Browsers/SO, great idea :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (helderburato) [reply .github/PULL_REQUEST_TEMPLATE.md]: I made an update with option to non-technical things, what do you think about?

**Review by vitormalencar — APPROVED** [review]
Amazing job 👍🏾 
thanks for taking care of this =)

**Review by ktfth — APPROVED** [review]
It's a good resource to cover the reports

**Pull request merged**